core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -103,6 +111,8 @@ export class TranslationModalComponent {
   textToTranslate: string | string[] = '';
   languageDescription: string;
   activeStatus: Status;
+  contentTab = ExpansionTabType.CONTENT;
+  translationTab = ExpansionTabType.TRANSLATION;
```

Comment:
Could you try [this](https://stackoverflow.com/a/44045373) approach so that we don't need to declare a separate variable for each enum value?

Reply:
Done